src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -3,27 +3,33 @@
             [clojure.string :as string]
             [oops.core :as oops]
             [quo.components.avatars.account-avatar.view :as account-avatar]
+            [quo.components.avatars.user-avatar.view :as user-avatar]
+            [quo.components.avatars.wallet-user-avatar.view :as wallet-user-avatar]
             [quo.components.buttons.button.view :as button]
-            [quo.components.gradient.gradient-cover.view :as gradient-cover]
```

Comment:
@mmilad75 this was correct this way. Please check the figma hierarchy

<img width="342" alt="Screenshot 2024-01-31 at 12 08 10" src="https://github.com/status-im/status-mobile/assets/22799766/fa33cafb-b979-4568-86e7-2cc36fc5c0a8">

Reply:
Done